src/Features/ERDDAP/types.ts

```diff
@@ -113,7 +114,7 @@ export interface PlatformProgram {
 
 export interface DataType {
   standard_name: string
-  short_name: string
+  short_name?: string
```

Comment:
Codacy found an issue: [Identifier 'short_name' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)